asdar/Makefile

```diff
@@ -13,6 +13,9 @@ TESTS_HEADERS := $(filter-out tests/runcalls.h, $(wildcard tests/*.h)) tests/run
 
 IWYU ?= iwyu
 
+ifdef NDEBUG
+	CFLAGS += -DNDEBUG
+endif
```

Comment:
Is a global debugging print switch really a good idea? What if I want to turn it off for file `foo.c` but turn it on for file `bar.c`? That's easy to do if each source file has its own debug defines.

Reply:
`NDEBUG` is a standard-ish thing that also turns off `assert`s

Reply:
I don't want to disable asserts just to get rid of debug prints in `runner.c`